game_view.rb

```diff
@@ -22,4 +25,33 @@ def clear_view
     system "clear"
   end
 
+  def display_hangman_head
+    "      " + "_____\n" + "     " + "( * * )\n" +  "      " + "(___)"
```

Comment:
For these body part methods it would be nice to line up the different rows on new lines so you can kind of see what the part will look like from just looking at the code